src/main.go

```diff
@@ -57,12 +52,8 @@ func configureEnvironment() {
 	viper.AutomaticEnv()
 
 	viper.SetDefault("ENV", "local") // local or production only
-	viper.SetDefault("GOFIRE_MASTER", false)
-	viper.SetDefault("GOFIRE_HOST", "127.0.0.1")
-	viper.SetDefault("GOFIRE_PORT", 8001)
-	viper.SetDefault("GOFIRE_MASTER_PORT", 8000)
-	viper.SetDefault("GOFIRE_MASTER_HOST", "127.0.0.1")
-	viper.SetDefault("GOFIRE_MOCK_GPIO", true)
+	viper.SetDefault("HOST", "0.0.0.0")
+	viper.SetDefault("PORT", 8001)
 	viper.SetDefault("CLUSTER_NAME", "MasterOfHot")
-	viper.SetDefault("MICROCONTORLLER_LIMIT", 255)
+	viper.SetDefault("SUBNET_ROOT", "192.16.1.")
```

Comment:
This is not a subnet. Default subnet looks like `255.255.255.0` and is different than the IP Address.

However I think handling multiple/different subnets could be added later. Once we start getting closer to the 255 device limit.

Reply:
What you are describing is a subnet _mask_. What I have there is a subnet _address_. I'll update the variable to be more descriptive. 
Alternatively, we could use a mask but we'll need to take a gateway address in as an env var too & apply the mask to it in `Start()`. I can do that if we need that flexibility, but as you said, we're not approaching 255 devices yet. 

```
Subnet - A portion of a network that shares a particular subnet address.

Subnet mask - A 32-bit combination used to describe which portion of an address refers to the subnet and which part refers to the host.
```

Reply:
```suggestion
	viper.SetDefault("IP_ADDRESS_ROOT", "192.16.1.")
```